scrapyd_k8s.sample-k8s.conf

```diff
@@ -57,3 +57,16 @@ requests_cpu = 80m
 requests_memory = 0.12G
 limits_cpu = 0.5
 limits_memory = 0.2G
+
+;[joblogs]
+;# Choose storage provider
+;storage_provider = s3
+;container_name   = <container_name>
+
+;# Choose number of unique logs, but at least 2
+;num_lines_to_check = 2
+
+;[joblogs.storage.s3]
+;key    = ${S3_KEY}
+;secret = ${S3_SECRET}
+;region = ${S3_REGION}
```

Comment:
Can use we `# ` to comment this out instead? `;` looks like it's fine, but it's less common to use for commenting out.
Also, let's include a real-world (commented-out) example here, so use a container_name like `scrapyd-k8s-example-bucket`, and explain in a comment with the storage, that one needs to set the `S3_KEY` and `S3_SECRET` environment variables. Please include `region`, so it's clear to readers that one can mix and match between environment and string, also in the storage.s3 section